src/applications/caregivers/tests/helpers.unit.spec.js

```diff
@@ -13,6 +13,7 @@ import requiredOnly from './e2e/fixtures/data/requiredOnly.json';
 import secondaryTwoOnly from './e2e/fixtures/data/secondaryOneOnly.json';
 import oneSecondaryCaregivers from './e2e/fixtures/data/oneSecondaryCaregivers.json';
 import twoSecondaryCaregivers from './e2e/fixtures/data/twoSecondaryCaregivers.json';
+import signAsRepresentativeNo from './e2e/fixtures/data/signAsRepresentativeNo.json';
```

Comment:
Should probably move this data up a level so its not in `e2e` since its shared between UT and e1e... tech debt for sure